src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -5,8 +5,7 @@
     <LangVersion>7.3</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <SuppressFinalPackageVersion>false</SuppressFinalPackageVersion>
-    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
-    <VersionPrefix>0.4.0</VersionPrefix>
+    <VersionPrefix>0.4.1</VersionPrefix>
```

Comment:
?? Why not `0.5.0`? We should reserve `0.4.1` in case we need to patch a bad bug in `0.4.0`.

Reply:
because this produces 0.4.1.alpha-something. When we want a stable 0.4.1, we just set `<DotNetFinalVersionKind>release</DotNetFinalVersionKind>` again. The problem with making it `0.5.0` is that we'd start producing `0.5.0.alpha-something` and if we ever make a `0.4.1`, we'd run into the minor issue that `0.4.1` will list below the older `0.5.0.alpha` packages.

Reply:
> we'd run into the minor issue that 0.4.1 will list below the older 0.5.0.alpha packages.

This is expected. If we ever made `0.4.1` it would be branched off of `0.4.0`, not `master`.

Read https://semver.org/ to understand when to bump a minor version and when to bump the patch., As we want to add more APIs/features, we want to bump the minor version.

Reply:
Oh, you're thinking about `0.4.1.alpha-` as an official built artifact too. I was thinking `0.4.0` is the current official build, and we'd still have an option to make a `0.4.1` or a `0.5.0` and have them list in the correct order. This is all just to get around the fact that we don't have official release branches in corefxlab. I changed the version to `0.5.0`. If you change your mind and want it back to `0.4.1`, let me know